.github/workflows/release.yml

```diff
@@ -0,0 +1,135 @@
+name: Release on Merge to Master
+
+on:
+  push:
+    branches:
+      - master
+
+permissions: write-all
```

Comment:
Would you please cope down the permissions the workflow needs?

Reply:
Fixed